recipes/go-precompiled/conda_build_config.yaml

```diff
@@ -0,0 +1,7 @@
+go_variant_ver:
+  - 1.1.1
+go_variant_str:
+  - precompiled
+MACOSX_DEPLOYMENT_TARGET:
+  - 10.11
```

Comment:
```suggestion
```

We are not passing any of the `cgo` tests, so there is no point in enforcing a MACOS_DEPLOYMENT_TARGET